content/ko/docs/helm/helm.md

```diff
@@ -0,0 +1,100 @@
+|------------------------------------|-----------------------------------------------------------------------------------|
+| $HELM_CACHE_HOME                    | 캐시된 파일을 저장할 대체 위치를 지정.                                                          |
+| $HELM_CONFIG_HOME                   | 헬름 설정을 저장할 대체 위치 지정.                                                          |
+| $HELM_DATA_HOME                     | 헬름 데이터를 저장할 대체 위치를 지정.                                                        |
+| $HELM_DRIVER                       | 백엔드 스토리지 드라이버 설정. 값 : configmap, secret, memory, postgres                       |
+| $HELM_DRIVER_SQL_CONNECTION_STRING | SQL 스토리지 드라이버가 사용해야 하는 연결 문자열 지정.                                              |
+| $HELM_NO_PLUGINS                   | 플러그인 비활성화. 비활성화 하기 위해 HELM_NO_PLUGINS=1 로 지정.                                  |
+| $KUBECONFIG                        | 대체 쿠버네티스 설정 파일 지정 (기본값 "~/.kube/config")                                              |
+
+헬름은 다음 설정 순서를 기반으로 캐시, 설정 정보, 데이터를 저장한다:
+
+- HOME_*_HOME 환경변수가 설정된 경우 이 변수를 사용함
+- 그렇지 않으면, XDG 기반 디렉토리 사양을 지원하는 시스템에서 XDG 변수를 사용함
+- 다른 위치를 설정하지 않은 경우 운영 체제의 기본 위치를 사용함
+
+기본적으로 기본 디렉토리는 운영체제에 따라 다르며, 기본 값은 아래와 같다.
+
+| 운영체제             | 캐시 파일 경로                 | 설정 파일 경로                      | 데이터 저장 경로              |
+|------------------|---------------------------|--------------------------------|-------------------------|
+| Linux            | $HOME/.cache/helm         | $HOME/.config/helm             | $HOME/.local/share/helm |
+| macOS            | $HOME/Library/Caches/helm | $HOME/Library/Preferences/helm | $HOME/Library/helm      |
+| Windows          | %TEMP%\helm               | %APPDATA%\helm                 | %APPDATA%\helm          |
+
+
+### 옵션 
+
+```
+      --add-dir-header                   이 값이 참이면, 헤더에 파일 디렉토리를 추가
+      --alsologtostderr                  표준 오류를 로그 및 파일로 표시
+      --debug                            상세 내용 표시 활성화
+  -h, --help                             헬름에 대한 도움말
+      --kube-apiserver string            쿠버네티스 API 서버의 주소 및 포트
+      --kube-context string              사용할 kubeconfig 컨텍스트 이름
+      --kube-token string                인증에 사용될 문자열 전달 토큰
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N 행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   이 값이 비어있지 않을 경우, 이 값에 지정된 디렉토리에 로그 파일 쓰기 수행
+      --log-file string                  이 값이 비어있지 않을 경우, 이 값에 지정된 파일명으로 로그 파일 쓰기 수행
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가 바이트이며 값이 0일 경우 최대 파일크기 제한 없음(기본 값 1800)
+      --logtostderr                      로그를 파일이 아닌 표준 출력으로 표시 (기본값 : true)
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
```

Comment:
Ah! This is where you were getting that. Well, I'll figure out why it is only in this one place in the English. No change for now.